apps/web/public/static/locales/en/common.json

```diff
@@ -2322,7 +2322,13 @@
   "success_entry_created": "Successfully created a new entry",
   "booking_redirect_email_subject": "Booking redirect notification",
   "booking_redirect_email_title": "Booking Redirect Notification",
-  "booking_redirect_email_description": "You have received a booking redirection from {{toName}} so their profile links will be redirect to yours for the time interval: ",
+  "booking_redirect_email_description": "You have received a booking redirection from {{toName}} so their profile links will be redirect to yours for the time interval: \"{{dates}}\"",
+  "booking_redirect_edit_email_subject": "Booking redirect edit notification",
+  "booking_redirect_edit_email_title": "Booking Redirect Edit Notification",
+  "booking_redirect_edit_email_description": "Your booking redirection from {{toName}} for interval \"{{oldDates}}\" have been edited. New time interval for the redirection is: \"{{dates}}\"",
```

Comment:
Not` edited,` it should be `updated` or `modified`.

Reply:
Done!